sql-plugin/src/main/spark350/scala/com/nvidia/spark/rapids/shims/PartitionedFileUtilsShim.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2023-2024, NVIDIA CORPORATION.
```

Comment:
Should this new file have a copyright from backdate?

Reply:
Ah, shoot. Will fix.